src/ActionInterface.cpp

```diff
@@ -366,11 +366,10 @@ void ActionInterface::OffsetStandbyFrequency(double offset_khz, bool to_devices)
 
 void ActionInterface::ExchangeRadioFrequencies(bool to_devices)
 {
-  const auto radio_settings = SetComputerSettings().radio;
-
-  const auto old_active_freq = radio_settings.active_frequency;
-  const auto old_active_freq_name = radio_settings.active_name;
+  /* send to external devices */
 
-  ActionInterface::SetActiveFrequency(radio_settings.standby_frequency, radio_settings.standby_name, to_devices);
-  ActionInterface::SetStandbyFrequency(old_active_freq, old_active_freq_name, to_devices);
+  if (to_devices) {
```

Comment:
You're not updating our own internal administration here, which appears incorrect to me. When is this updated?
Previously it happened because as a result of the call to `SetActiveFrequency` and `SetStandbyFrequency`.
I can't comment on the other devices, but the KRT2 driver doesn't give any feedback to XCSoar after exchanging the frequencies.
I suspect this code would cause the radio and xcsoar (in the infoboxes I added) to display two different things.

Reply:
Yes, I saw that as well in the documentation I received. 

PR is WIP (it was marked as such, don't know who removed it).